<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>689</width>
    <height>442</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Editor de botones</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>images/editor/text.png</normaloff>images/editor/text.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <widget class="QPushButton" name="b_accept">
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>300</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>De acuerdo</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>images/ok.png</normaloff>images/ok.png</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QListWidget" name="listWidget">
     <property name="minimumSize">
      <size>
       <width>240</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>406</height>
      </size>
     </property>
     <property name="palette">
      <palette>
       <active>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>195</red>
           <green>218</green>
           <blue>224</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>195</red>
           <green>217</green>
           <blue>224</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>195</red>
           <green>218</green>
           <blue>224</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>195</red>
           <green>217</green>
           <blue>224</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>195</red>
           <green>217</green>
           <blue>224</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>195</red>
           <green>217</green>
           <blue>224</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="toolTip">
      <string>hola</string>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="alternatingRowColors">
      <bool>false</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::NoSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="viewMode">
      <enum>QListView::ListMode</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QPushButton" name="b_cancel">
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>300</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Cancelar</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>images/cancel.png</normaloff>images/cancel.png</iconset>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
